core/plugins/org.polarsys.capella.core.ui.properties.richtext/src/org/polarsys/capella/core/ui/properties/richtext/sections/CapellaDescriptionPropertySection.java

```diff
@@ -120,11 +120,13 @@ public IStatus runInUIThread(IProgressMonitor monitor) {
    */
   @Override
   public void loadData(EObject descriptorOrCapellaElement) {
-    super.loadData(descriptorOrCapellaElement);
-    mapDescriptionSectionToEObject.put(CapellaDescriptionPropertySection.this, descriptorOrCapellaElement);
+    if (descriptorOrCapellaElement != null && !descriptorOrCapellaElement.equals(job.current)) {
+      super.loadData(descriptorOrCapellaElement);
```

Comment:
I think the super.loadData shall be outside the if.
https://github.com/eclipse/capella/commit/c52365a209b226c4f3888e77aae32a841601f93f#diff-0481f92d1e0406a7269ab3ad3a5042d0018ead747aa11f18c97b231dd193a890R272